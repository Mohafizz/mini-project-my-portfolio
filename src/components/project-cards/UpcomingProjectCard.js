import React from "react";
import {
  CardTitle,
  Card,
  CardText,
  CardActions,
  Button,
  Grid,
  Cell
} from "react-mdl";

const UpcomingProjectCard = () => {
  return (
    <div>
      <div style={{ width: "100%", margin: "auto" }}>
        <Grid className="projects-grid">
          <Cell col={4}>
            <Card
              shadow={0}
              style={{ width: "320px", height: "320px", margin: "auto" }}
            >
              <CardTitle
                expand
                style={{
                  color: "#fff",
                  background: "url(Images/wip.png) center no-repeat #ffffff",
                  backgroundSize: "200px"
                }}
              />
              <CardText>Upcoming project. More to go!</CardText>
              <CardActions border>
                <Button colored>View Updates</Button>
              </CardActions>
            </Card>
          </Cell>
        </Grid>
      </div>
    </div>
  );
};

export default UpcomingProjectCard;
