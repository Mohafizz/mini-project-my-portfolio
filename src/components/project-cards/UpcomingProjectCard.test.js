import React from "react";
import { shallow } from "enzyme";
import UpcomingProjectCard from "./UpcomingProjectCard";

const upcomingCard = shallow(<UpcomingProjectCard />);

it("should render correctly", () => {
  expect(upcomingCard).toMatchSnapshot();
});
