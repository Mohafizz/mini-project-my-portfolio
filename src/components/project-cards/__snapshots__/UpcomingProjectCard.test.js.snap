// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UpcomingProjectCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        style={
                Object {
                        "margin": "auto",
                        "width": "100%",
                      }
        }
>
        <Grid
                className="projects-grid"
        >
                <Cell
                        col={4}
                >
                        <Card
                                shadow={0}
                                style={
                                        Object {
                                                "height": "320px",
                                                "margin": "auto",
                                                "width": "320px",
                                              }
                                }
                        >
                                <CardTitle
                                        expand={true}
                                        style={
                                                Object {
                                                        "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                        "backgroundSize": "200px",
                                                        "color": "#fff",
                                                      }
                                        }
                                />
                                <CardText>
                                        Upcoming project. More to come! 
                                        <br />
                                         Click button below to view my github repo.
                                </CardText>
                                <CardActions
                                        border={true}
                                >
                                        <Button
                                                accent={true}
                                                href="https://github.com/Mohafizz"
                                                raised={true}
                                                target="blank"
                                        >
                                                My github repo
                                        </Button>
                                </CardActions>
                        </Card>
                </Cell>
        </Grid>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Grid
          className="projects-grid"
>
          <Cell
                    col={4}
          >
                    <Card
                              shadow={0}
                              style={
                                        Object {
                                                  "height": "320px",
                                                  "margin": "auto",
                                                  "width": "320px",
                                                }
                              }
                    >
                              <CardTitle
                                        expand={true}
                                        style={
                                                  Object {
                                                            "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                            "backgroundSize": "200px",
                                                            "color": "#fff",
                                                          }
                                        }
                              />
                              <CardText>
                                        Upcoming project. More to come! 
                                        <br />
                                         Click button below to view my github repo.
                              </CardText>
                              <CardActions
                                        border={true}
                              >
                                        <Button
                                                  accent={true}
                                                  href="https://github.com/Mohafizz"
                                                  raised={true}
                                                  target="blank"
                                        >
                                                  My github repo
                                        </Button>
                              </CardActions>
                    </Card>
          </Cell>
</Grid>,
        "style": Object {
          "margin": "auto",
          "width": "100%",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Cell
            col={4}
>
            <Card
                        shadow={0}
                        style={
                                    Object {
                                                "height": "320px",
                                                "margin": "auto",
                                                "width": "320px",
                                              }
                        }
            >
                        <CardTitle
                                    expand={true}
                                    style={
                                                Object {
                                                            "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                            "backgroundSize": "200px",
                                                            "color": "#fff",
                                                          }
                                    }
                        />
                        <CardText>
                                    Upcoming project. More to come! 
                                    <br />
                                     Click button below to view my github repo.
                        </CardText>
                        <CardActions
                                    border={true}
                        >
                                    <Button
                                                accent={true}
                                                href="https://github.com/Mohafizz"
                                                raised={true}
                                                target="blank"
                                    >
                                                My github repo
                                    </Button>
                        </CardActions>
            </Card>
</Cell>,
          "className": "projects-grid",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Card
              shadow={0}
              style={
                            Object {
                                          "height": "320px",
                                          "margin": "auto",
                                          "width": "320px",
                                        }
              }
>
              <CardTitle
                            expand={true}
                            style={
                                          Object {
                                                        "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                        "backgroundSize": "200px",
                                                        "color": "#fff",
                                                      }
                            }
              />
              <CardText>
                            Upcoming project. More to come! 
                            <br />
                             Click button below to view my github repo.
              </CardText>
              <CardActions
                            border={true}
              >
                            <Button
                                          accent={true}
                                          href="https://github.com/Mohafizz"
                                          raised={true}
                                          target="blank"
                            >
                                          My github repo
                            </Button>
              </CardActions>
</Card>,
            "col": 4,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <CardTitle
                  expand={true}
                  style={
                                    Object {
                                                      "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                      "backgroundSize": "200px",
                                                      "color": "#fff",
                                                    }
                  }
/>,
                <CardText>
                  Upcoming project. More to come! 
                  <br />
                   Click button below to view my github repo.
</CardText>,
                <CardActions
                  border={true}
>
                  <Button
                                    accent={true}
                                    href="https://github.com/Mohafizz"
                                    raised={true}
                                    target="blank"
                  >
                                    My github repo
                  </Button>
</CardActions>,
              ],
              "shadow": 0,
              "style": Object {
                "height": "320px",
                "margin": "auto",
                "width": "320px",
              },
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "expand": true,
                  "style": Object {
                    "background": "url(Images/wip.png) center no-repeat #ffffff",
                    "backgroundSize": "200px",
                    "color": "#fff",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    "Upcoming project. More to come! ",
                    <br />,
                    " Click button below to view my github repo.",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "Upcoming project. More to come! ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": "br",
                  },
                  " Click button below to view my github repo.",
                ],
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "border": true,
                  "children": <Button
                    accent={true}
                    href="https://github.com/Mohafizz"
                    raised={true}
                    target="blank"
>
                    My github repo
</Button>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "accent": true,
                    "children": "My github repo",
                    "href": "https://github.com/Mohafizz",
                    "raised": true,
                    "target": "blank",
                  },
                  "ref": null,
                  "rendered": "My github repo",
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          style={
                    Object {
                              "margin": "auto",
                              "width": "100%",
                            }
          }
>
          <Grid
                    className="projects-grid"
          >
                    <Cell
                              col={4}
                    >
                              <Card
                                        shadow={0}
                                        style={
                                                  Object {
                                                            "height": "320px",
                                                            "margin": "auto",
                                                            "width": "320px",
                                                          }
                                        }
                              >
                                        <CardTitle
                                                  expand={true}
                                                  style={
                                                            Object {
                                                                      "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                                      "backgroundSize": "200px",
                                                                      "color": "#fff",
                                                                    }
                                                  }
                                        />
                                        <CardText>
                                                  Upcoming project. More to come! 
                                                  <br />
                                                   Click button below to view my github repo.
                                        </CardText>
                                        <CardActions
                                                  border={true}
                                        >
                                                  <Button
                                                            accent={true}
                                                            href="https://github.com/Mohafizz"
                                                            raised={true}
                                                            target="blank"
                                                  >
                                                            My github repo
                                                  </Button>
                                        </CardActions>
                              </Card>
                    </Cell>
          </Grid>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Grid
            className="projects-grid"
>
            <Cell
                        col={4}
            >
                        <Card
                                    shadow={0}
                                    style={
                                                Object {
                                                            "height": "320px",
                                                            "margin": "auto",
                                                            "width": "320px",
                                                          }
                                    }
                        >
                                    <CardTitle
                                                expand={true}
                                                style={
                                                            Object {
                                                                        "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                                        "backgroundSize": "200px",
                                                                        "color": "#fff",
                                                                      }
                                                }
                                    />
                                    <CardText>
                                                Upcoming project. More to come! 
                                                <br />
                                                 Click button below to view my github repo.
                                    </CardText>
                                    <CardActions
                                                border={true}
                                    >
                                                <Button
                                                            accent={true}
                                                            href="https://github.com/Mohafizz"
                                                            raised={true}
                                                            target="blank"
                                                >
                                                            My github repo
                                                </Button>
                                    </CardActions>
                        </Card>
            </Cell>
</Grid>,
          "style": Object {
            "margin": "auto",
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Cell
              col={4}
>
              <Card
                            shadow={0}
                            style={
                                          Object {
                                                        "height": "320px",
                                                        "margin": "auto",
                                                        "width": "320px",
                                                      }
                            }
              >
                            <CardTitle
                                          expand={true}
                                          style={
                                                        Object {
                                                                      "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                                      "backgroundSize": "200px",
                                                                      "color": "#fff",
                                                                    }
                                          }
                            />
                            <CardText>
                                          Upcoming project. More to come! 
                                          <br />
                                           Click button below to view my github repo.
                            </CardText>
                            <CardActions
                                          border={true}
                            >
                                          <Button
                                                        accent={true}
                                                        href="https://github.com/Mohafizz"
                                                        raised={true}
                                                        target="blank"
                                          >
                                                        My github repo
                                          </Button>
                            </CardActions>
              </Card>
</Cell>,
            "className": "projects-grid",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Card
                shadow={0}
                style={
                                Object {
                                                "height": "320px",
                                                "margin": "auto",
                                                "width": "320px",
                                              }
                }
>
                <CardTitle
                                expand={true}
                                style={
                                                Object {
                                                                "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                                "backgroundSize": "200px",
                                                                "color": "#fff",
                                                              }
                                }
                />
                <CardText>
                                Upcoming project. More to come! 
                                <br />
                                 Click button below to view my github repo.
                </CardText>
                <CardActions
                                border={true}
                >
                                <Button
                                                accent={true}
                                                href="https://github.com/Mohafizz"
                                                raised={true}
                                                target="blank"
                                >
                                                My github repo
                                </Button>
                </CardActions>
</Card>,
              "col": 4,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <CardTitle
                    expand={true}
                    style={
                                        Object {
                                                            "background": "url(Images/wip.png) center no-repeat #ffffff",
                                                            "backgroundSize": "200px",
                                                            "color": "#fff",
                                                          }
                    }
/>,
                  <CardText>
                    Upcoming project. More to come! 
                    <br />
                     Click button below to view my github repo.
</CardText>,
                  <CardActions
                    border={true}
>
                    <Button
                                        accent={true}
                                        href="https://github.com/Mohafizz"
                                        raised={true}
                                        target="blank"
                    >
                                        My github repo
                    </Button>
</CardActions>,
                ],
                "shadow": 0,
                "style": Object {
                  "height": "320px",
                  "margin": "auto",
                  "width": "320px",
                },
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "expand": true,
                    "style": Object {
                      "background": "url(Images/wip.png) center no-repeat #ffffff",
                      "backgroundSize": "200px",
                      "color": "#fff",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": Array [
                      "Upcoming project. More to come! ",
                      <br />,
                      " Click button below to view my github repo.",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    "Upcoming project. More to come! ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": "br",
                    },
                    " Click button below to view my github repo.",
                  ],
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "border": true,
                    "children": <Button
                      accent={true}
                      href="https://github.com/Mohafizz"
                      raised={true}
                      target="blank"
>
                      My github repo
</Button>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "accent": true,
                      "children": "My github repo",
                      "href": "https://github.com/Mohafizz",
                      "raised": true,
                      "target": "blank",
                    },
                    "ref": null,
                    "rendered": "My github repo",
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
